components/comparision/ComparisonToolHero.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <div>
+    <div
+      class="hero d-flex justify-content-center align-items-center position-relative mx-auto"
+    >
+      <div
+        class="hero__content d-flex flex-column align-items-center my-auto text-center px-3"
+      >
+        <span class="hero__subtitle">Comparison Tool</span>
+        <h1 class="hero__heading">
+          Confused? Easy way to compare Form Builders
+        </h1>
+        <p class="hero__description mt-3">
+          Compare Formester with other Form Building tools.
+        </p>
+        <a
+          href="https://app.formester.com/users/sign_up"
+          class="btn button hero__button mt-4 mb-4 mb-lg-0"
+          >Sign Up For Free</a
+        >
+      </div>
+      <div>
+        <transition name="slide-in-left">
+          <nuxt-img
+            src="comparision/comparison-left-hero-image.svg"
+            alt="Hero-Image"
+            class="img-fluid position-absolute hero-image left"
+            v-show="showImage"
+          />
+        </transition>
+        <transition name="slide-in-right">
+          <nuxt-img
+            src="comparision/comparison-right-hero-image.svg"
+            alt="Hero-Image"
+            class="img-fluid position-absolute hero-image right"
+            v-show="showImage"
+          />
+        </transition>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      showImage: false,
+    }
+  },
+  mounted() {
+    this.showImage = true
+  },
+}
+</script>
+
+<style scoped>
+.hero {
+  margin-top: 130px;
+  margin-bottom: 100px;
+  max-width: 1440px;
+}
+
+.hero__content {
+  max-width: 820px;
+}
+
+.hero__subtitle {
+  color: var(--clr-primary);
+  font-size: 16px;
+  font-weight: 600;
```

Comment:
variables for font-size and weight

Reply:
we have some style for font-size that was created initially. Let's see and replace it.